### ISSUE TITLE
Wrong column number in 0.4.3

### ISSUE THREAD

**Issue by iamvdo** [mozilla/source-map#192]
As #191, I also get errors with 0.4.3 from PostCSS sourcemap.

It returns wrong column from `generatedPositionFor`.

I have this file `module.css`:
```css
.module {
  color: #fff
}
```
And the processed file:
```css
.import{color:#fff}.module{color:#fff}.main{color:#fff}
```

When I use :
```javascript
var smc = new SourceMap.SourceMapConsumer(PostCSS_map);
var positions = smc.generatedPositionFor({source: 'module.css', line: 1, column: 0});
```
`positions.column == 19` with @0.4.2 and `positions.column == 27` with @0.4.3

Is it intended? Am I missing something?

**Comment by fitzgen** [comment]
Can you package up the test case into a gist and give me steps to reproduce with the packaged test case? That will help me debug the issue. Thanks!

**Comment by iamvdo** [comment]
Well, here you go: https://gist.github.com/iamvdo/7051563e4383dee93da8

Install modules
```
npm i postcss csswring source-map
```

Then switch between `source-map@0.4.2` and `source-map@0.4.3` from `node_modules/postcss` folder.

With `source-map@0.4.2`, you get mappings for `module.css`:
```
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 19,
  originalLine: 1,
  originalColumn: 0,
  name: null }
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 27,
  originalLine: 2,
  originalColumn: 2,
  name: null }
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 37,
  originalLine: 2,
  originalColumn: 14,
  name: null }
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 38,
  originalLine: 3,
  originalColumn: 1,
  name: null }
```
While with `source-map@0.4.3`, you get:
```
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 27,
  originalLine: 2,
  originalColumn: 2,
  name: null }
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 37,
  originalLine: 2,
  originalColumn: 14,
  name: null }
{ source: 'module.css',
  generatedLine: 1,
  generatedColumn: 38,
  originalLine: 3,
  originalColumn: 1,
  name: null }
```

So, the generated position is `{ line: 1, column: 19, lastColumn: null }` in case one while it's `{ line: null, column: null, lastColumn: null }`in case two.

**Comment by iamvdo** [comment]
The position `27` from the first message is definitely because I use many PostCSS plugins.

**Comment by fitzgen** [comment]
Thanks for the test case. I have a failing regression test and am looking into what's up now.

**Comment by fitzgen** [comment]
Hm actually my regression test isn't sufficient.

**Comment by fitzgen** [comment]
Fixed in 9b53b00

**Comment by iamvdo** [comment]
:+1:
